### PR TITLE
feat: add `primereact` example

### PR DESCRIPTION
**Pull request by kendallcamposs**

### CODE REVIEW

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]
Please update this example to start from the `__template` folder & add the rest of your example code afterwards

File: primereact/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+```sh
+npm start
+```
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/frosty-christian-7yu236?file=%2FREADME.md)
```
Comment (MichaelDeBoey) [inline primereact/README.md:37]:
```suggestion
[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/examples/tree/main/primereact)
```

File: primereact/app/entry.client.tsx
Comment (MichaelDeBoey) [inline primereact/app/entry.client.tsx:1]: Please remove this file

File: primereact/app/entry.server.tsx
Comment (MichaelDeBoey) [inline primereact/app/entry.server.tsx:1]: Please remove this file

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by machour** [comment]
Hey @kendallcamposs 👋🏼 
Is this ready for review or still a work in progress?

**Comment by kendallcamposs** [comment]
Hi @machour!

I think we should close it because of this issue: https://github.com/primefaces/primereact/issues/4675

**Comment by AfzalSabbir** [comment]
Stuck here as well.
As it's a dead end, and my project is in a begine stage. I'm going to move to something else.

**Pull request closed without merging**